src/fibonacciBases.js

```diff
@@ -0,0 +1,89 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibonacci = []
+
+//calculate fibonacci value
+  function fibonacciValue(resultArr) {
+    let value = 0
+    for(let i = 0; i < fibonacci.length; i++) {
+      value += ( resultArr[i] * fibonacci[i] )
+    } return value
+  }
+  // why doesnt this work?
+// const fibonacciValue = resultArr => {
+//    [0,1,2,3,4,5].reduce(total, currentValue, currentIndex) => total + current * fibonacci[currentIndex] }
```

Comment:
This line is just a syntax error, reduce takes 2 arguments, first is the function to perform, the second is the accumulator object. You were close, you gave the function inside reduce the right arguments, but I didn't see the accumulator itself provided
[0,1, 2, 3, 4, 5].reduce( ()=>{}, accumulator )
[0,1, 2, 3, 4, 5].reduce( (accumulator, currentValue, currentIndex)=>{}, accumulatorObject )